keps/sig-api-machinery/2885-server-side-unknown-field-validation/README.md

```diff
@@ -874,46 +869,29 @@ Think about adding additional work or introducing new steps in between
 
 [existing SLIs/SLOs]: https://git.k8s.io/community/sig-scalability/slos/slos.md#kubernetes-slisslos
 -->
-Mutating API calls that opt-in to validation will be slower ([initial
-benchmarks](https://github.com/kubernetes/kubernetes/pull/104433#issuecomment-901398507)
-estimate ~20% slower 25-30% more memory consumption)
+Mutating API calls that opt-in to validation will be slower.
+Benchmarks of alpha changes indicate that performing validation results in
+~2-5% slower request latency and ~4-8% more memory usage.
 
-<!--
 ###### Will enabling / using this feature result in non-negligible increase of resource usage (CPU, RAM, disk, IO, ...) in any components?
 
-Things to keep in mind include: additional in-memory state, additional
-non-trivial computations, excessive access to disks (including increased log
-volume), significant amount of data sent and/or received over network, etc.
-This through this both in small and large cases, again with respect to the
-[supported limits].
-
-[supported limits]: https://git.k8s.io/community//sig-scalability/configs-and-limits/thresholds.md
+Non-negligible increase to RAM of kube-apiserver when server-side validation is
+heavily utilized.
```

Comment:
4-8% increase in mutating json/yaml requests is not nothing, but is also a much smaller percentage of overall resource usage... I think overall this is actually negligible

Reply:
done